### ISSUE TITLE
One thing about the normalization 

### ISSUE THREAD

**Issue by tonghe90** [zhmiao/OpenLongTailRecognition-OLTR#26]
Thanks for sharing the code. I have one question about the squashing function + cross entropy loss. Do you have some experiments about using softmax + cross entropy loss? Or other normalization method

**Comment by zhmiao** [comment]
Hello @tonghe90 , thanks for asking. Yes, we have tested pure softmax + cross entropy and simple cosine normalization + cross entropy. Neither of them were as good as  squashing because squashing is relatively more relaxed compared to cosine, and normalized than direct feature. We are sorry wer did not report the comparison in the paper due to space limitations.
